co2eq/food/meal.js

```diff
@@ -0,0 +1,73 @@
+import {
+  MEAL_TYPE_VEGAN,
+  MEAL_TYPE_VEGETARIAN,
+  MEAL_TYPE_PESCETARIAN,
+  MEAL_TYPE_MEAT_LOW,
+  MEAL_TYPE_MEAT_MEDIUM,
+  MEAL_TYPE_MEAT_HIGH,
+  MEAL_TYPE_MEAT_OR_FISH,
+} from '../../definitions';
+
+const MEALS_PER_DAY = 3;
+
+// ** modelName must not be changed. If changed then old activities will not be re-calculated **
+export const modelName = 'meal';
+export const modelVersion = '6'; // This model relies on footprints.yaml
+export const explanation = {
+  text: 'The calculations take into consideration greenhouse gas emissions across the whole lifecycle for an average meal of a specific diet.',
+  links: [
+    { label: 'Nature (2017)', href: 'https://www.nature.com/articles/s41598-017-06466-8' },
+    { label: 'Tomorrow footprint database', href: 'https://github.com/tmrowco/northapp-contrib/blob/master/co2eq/purchase/footprints.yml' },
```

Comment:
not sure this is applicable anymore

Reply:
@martincollignon just updated that yesterday if I'm right